### DISCUSSION TITLE
Welcome to marsupial-emr Discussions!

### DISCUSSION THREAD

**Discussion by marsupialproject** [marsupialproject/marsupial-emr#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Roentgeno** [comment]
First off, I want to say, your program is truly outstanding. Thank you for creating and sharing such a valuable tool!

I had a quick question regarding accessibility. Specifically, is the program compatible with screen readers and other assistive technologies? I’m hoping to incorporate it into student assignments and want to ensure that it’s fully accessible to all learners, including those who rely on screen readers.

Thanks so much for your time, and I appreciate any information or guidance you can share!

Warm regards,
Ryan

**Reply by marsupialproject** [reply]
Honestly, I'm not sure. If you can test out that functionality I would appreciate it 😄

**Reply by Roentgeno** [reply]
I am in the process of having my college look into the accessability and will let you know as soon as I find something out. Fingers-crossed! This is an incredible program!

**Comment by simtechd** [comment]
Hello! 

I'm currently integrating Marsupial into our sim lab and in the process of creating patient charts I've been utilizing the state function (awesome by the way). However, I'd really like to use the date advance feature and I've been unable to figure it out. I found two places where you reference a video about the advanced state functions but I have been unable to find that video - any help or breakdown on how the date advance works within the charts date system? (for context, we have three simulations that use the same patient - every sim is one day apart from the last)

Thank you!

**Reply by marsupialproject** [reply]
Sorry, forgot to make that video! It's pretty simple to use though. Just add an Advance Date function to whatever state you want and set the number of days to advance. Then set the date for any future orders/labs/etc as a negative value. When you hit that state, it will add the number of days to everything in the chart. For example, if you had the Advance Date set to 3, and had an order with a date set to -3, it will now have a date of 0 and show as today. Again, this will affect every date in the chart so it will appear as if 3 days have passed. The EMR is programmed to not show anything with a negative date, so you should be able to have multiple Advance Dates within the same patient.

If you have any other questions, please make a new topic and do not respond to the Welcome post.
